custom_components/plex_assistant/localize.py

```diff
@@ -121,6 +121,110 @@
             ],
         },
     },
+    "sv": {
+        "play": "spela",
+        "on_the": "på",
+        "not_found": "not found",
```

Comment:
"not_found" is used for errors like "episode not found" or "cast device not found". Should this be translated too?

Reply:
```suggestion
        "not_found": "finns ej",
```

Would be the right translation.